### ISSUE TITLE
rendering quoliti & upscaling setting

### ISSUE THREAD

**Issue by GyomeiHimejima9906** [iris-connect/iris-client#973]
**Is your feature request related to a problem? Please describe.**
some shaders pack are slow on iris becuase require more computional power from GPU

**Describe the solution you'd like**
possibility to manager rendiring quoliti and other rendering setting (like optifine that allow you to set: rendering quolity, normal and specular mapping for all shader packs)

**Describe alternatives you've considered**
yse, using optifine and don't using iris  (i corrently using and i conntinue tu use iris)

**Additional context**
https://github.com/MCRcortex/nvidium
![Screenshot 2024-03-10 140424](https://github.com/iris-connect/iris-client/assets/129201282/a90a9b7c-8d88-4048-b32f-87804c4fd565)

P.S. if possible can use vulkan for rendering? plase, thanks

**Comment by IMS212** [comment]
um. This isn't the right Iris.

**Comment by GyomeiHimejima9906** [comment]
no, this image i refferring a optifine that i suggest tu add to isi
but on iris the setting don't ere.
![image](https://github.com/iris-connect/iris-client/assets/129201282/63b2a539-df34-4a69-8040-503696ae54ce)
iris can't use an "upscaling" like Optifine
